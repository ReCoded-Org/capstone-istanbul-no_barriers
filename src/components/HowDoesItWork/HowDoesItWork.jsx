import React from "react";

const HowDoesItWork = () => {
    return <div></div>
};

export default HowDoesItWork