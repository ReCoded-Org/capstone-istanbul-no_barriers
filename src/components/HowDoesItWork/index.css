.howDoesItWork {
  padding: 2rem;
}

.howDoesItWork img {
  height: 7.25rem;
}
