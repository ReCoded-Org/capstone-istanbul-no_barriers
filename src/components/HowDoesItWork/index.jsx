import React from "react";
import Container from "react-bootstrap/Container";
import Row from "react-bootstrap/Row";
import Col from "react-bootstrap/Col";
import { useTranslation } from "react-i18next";
import NGO from "./NGO.png";
import preference from "./preference.png";
import service from "./service.png";
import "./index.css";

const HowDoesItWork = () => {
  const { t } = useTranslation();

  return (
    <Container fluid className="howDoesItWork">
      <h4>{t("homePage.howDoesItWork.title")}</h4>
      <br />
      <Row>
        <Col>
          <header>1</header>
          <br />
          <img src={preference} alt="preference" title="preference" />
          <div>{t("homePage.howDoesItWork.preference")}</div>
        </Col>
        <Col>
          <header>2</header>
          <br />
          <img src={service} alt="service" title="service" />
          <div>{t("homePage.howDoesItWork.service")}</div>
        </Col>
        <Col>
          <header>3</header>
          <br />
          <img src={NGO} alt="NGO" title="NGO" />
          <div>{t("homePage.howDoesItWork.connect")}</div>
        </Col>
      </Row>
    </Container>
  );
};

export default HowDoesItWork;
