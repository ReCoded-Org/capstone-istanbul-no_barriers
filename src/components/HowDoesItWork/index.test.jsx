import React from "react";
import renderer from "react-test-renderer";
import HowDoesItWork from "./index.jsx";

it("renders HowDoesItWork correctly", () => {
  const tree = renderer.create(<HowDoesItWork />).toJSON();
  expect(tree).toMatchSnapshot();
});
