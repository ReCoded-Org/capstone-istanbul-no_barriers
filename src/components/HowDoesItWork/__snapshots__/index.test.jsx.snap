// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders HowDoesItWork correctly 1`] = `
<div
  className="howDoesItWork container-fluid"
>
  <h4>
    homePage.howDoesItWork.title
  </h4>
  <br />
  <div
    className="row"
  >
    <div
      className="col"
    >
      <header>
        1
      </header>
      <br />
      <img
        alt="preference"
        src="preference.png"
        title="preference"
      />
      <div>
        homePage.howDoesItWork.preference
      </div>
    </div>
    <div
      className="col"
    >
      <header>
        2
      </header>
      <br />
      <img
        alt="service"
        src="service.png"
        title="service"
      />
      <div>
        homePage.howDoesItWork.service
      </div>
    </div>
    <div
      className="col"
    >
      <header>
        3
      </header>
      <br />
      <img
        alt="NGO"
        src="NGO.png"
        title="NGO"
      />
      <div>
        homePage.howDoesItWork.connect
      </div>
    </div>
  </div>
</div>
`;
